src/components/Tabs/components/Tab/Tab.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled, { css } from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import Element from '../../../Element';
+
+import { textMega } from '../../../../styles/style-helpers';
+
+const defaultTabStyles = ({ theme }) => css`
+  label: tab;
+  ${textMega({ theme })};
+  padding: ${theme.spacings.kilo} ${theme.spacings.tera};
+  color: ${theme.colors.n500};
+  text-decoration: none;
+  cursor: pointer;
+  background-color: transparent;
+  border: none;
+
+  white-space: nowrap;
+  height: 100%;
+  align-items: center;
+  float: left;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  outline: none;
+`;
+
+const selectedTabStyles = ({ theme, selected }) =>
+  selected &&
+  css`
+    label: tab--selected;
+    color: initial !important;
```

Comment:
shouldn't specificity handle this without the important? (just curiosity)

Reply:
I think this is leftover from the dashboard version where I needed to kill the default `Link` styles. Thanks!